exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -288,6 +296,7 @@ func (prwe *prwExporter) execute(ctx context.Context, writeReq *prompb.WriteRequ
 	}
 
 	if err != nil {
+		// prwe.settings.Logger.Error("error sending metrics: ", zap.Error(err))
```

Comment:
Should this line be removed, or just uncommented?

Reply:
removed it